draft-mouris-cfrg-mastic.md

```diff
@@ -661,15 +680,154 @@ malicious Aggregator.
 
 > NOTE to be specified in full detail.
 
+# Applications of `Mastic`
+
+Based on the two modes of operation of Mastic (i.e., Weighted Heavy-Hitters
+{{weighted-heavy-hitters}} and Attribute-Based Metrics
+{{aggregation-by-labels}}), we describe two applications of interest.
+
+## Network Error Logging {#NEL}
+Network Error Logging (NEL) is a mechanism used by web browsers to report errors
+that occur while attempting to establish a connection to a server {{W3C23}}. Some
+of these errors are visible to the server, but not all: failures in DNS, TCP,
+TLS, and HTTP can occur without the server having any visibility into the issue.
+A small amount of connection errors is expected, even under normal operating
+conditions; but a sudden, substantial increase in errors may be an indication of
+an outage, or a configuration issue impacting millions of users. Without a
+reporting mechanism like NEL, these events would only manifest in the server's
+telemetry as a drop in overall traffic.
+
+NEL is particularly important for content delivery networks, such as Akamai,
+AWS, or Cloudflare, that handle HTTP traffic for a large number of websites
+(typically millions). A content delivery network acts as a reverse proxy between
+clients and origin servers that provides a layer of caching and security
+services, such as DDoS protection.
+
+Reports are comprised of the URL the client attempted to navigate to (e.g.,
+`https://example.com`), the type of error that occurred, and metadata related to
+the attempt, such as the time that elapsed between when the connection attempt
+began and the error was observed (e.g., {{W3C23}}, Section 7). Clients may also
+report successful connection attempts to give the server a sense of the error
+rate. The exact client behavior is determined by the reporting policy specified
+by the server (see {{W3C23}}, Section 5.1).
+
+NEL data is privacy-sensitive for two reasons. First, it exposes information
+that the server would not otherwise have access to, which can be abused to probe
+the client's network configuration as described in [{{W3C23}}, Section 9].
+Second, for operational reasons, the reporting endpoint may be organizationally
+separated from the server (i.e., run on different cloud infrastructures),
+leading to an increased risk of the client's browsing history being exposed
+(e.g., in a data breach).
+
+**Private NEL with Mastic.** MPC helps mitigate these risks by revealing to the
```

Comment:
**This doesn't render the way you want it to in all version of internet drafts**.

```suggestion
MPC helps mitigate these risks by revealing to the
```